bigquery_to_datadog.py

```diff
@@ -0,0 +1,175 @@
+import google.cloud.bigquery as bigquery
+from datadog import initialize, api
+import time
+import json
+import os
+import logging
+
+
+logging.basicConfig(
+    filename="bigquery_to_dd.log",
+    level=logging.INFO,
+    format="%(asctime)s - %(levelname)s - %(message)s",
+)
+
+
+with open("config.json", "r") as config_file:
+    config = json.load(config_file)
+
+START_TIMESTAMP = "2024-06-18 22:05:00"
+# Configuration
+QUERIES = [
+    {
+        "name": "short_term_order_latency",
+        "query": """
+                SELECT p.received_at
+                    , TIMESTAMP_DIFF(p.received_at, s.sent_at, millisecond) AS latency
+                    , p.server_address
+                FROM `dydx-ce5e3.latency_experiments.long_running_two_sided_orders` s
```

Comment:
should any of this come from the config? not sure how the bq table names work

Reply:
moved project id into config